xlators/cluster/dht/src/dht-common.h

```diff
@@ -512,6 +513,20 @@ struct dht_methods_s {
 
 typedef struct dht_methods_s dht_methods_t;
 
+typedef struct {
+    gf_dirent_t orig_entries;
+    int op_errno;
+    dict_t *xdata;
+    int op_ret;
+    xlator_t *xl;
+    int ready;
+    gf_boolean_t in_progress;
```

Comment:
nit: if you are already creating a new structure, can we ensure it's aligned?

Reply:
Sure, I will do that in the next patch set.